example/lib/main.dart

```diff
@@ -125,11 +125,14 @@ class _MyAppState extends State<MyApp> {
         androidShowAnnotationListAction: true,
         showPageNumberOverlay: false,
         showPageLabels: true,
+        showDocumentTitle: false,
         invertColors: false,
         grayScale: false,
         startPage: 2,
         enableAnnotationEditing: true,
         enableTextSelection: false,
+        enableBookmarkList: false,
```

Comment:
I'd say we should enable bookmark list and document editor in the example by default (i.e. set these configuration options in the example to `true`, to make these features easier discoverable by trial user).

Reply:
I was thinking in terms of like: this is enabled by default (in the basic example as well), so the custom configuration shows how you can disable them. But if you think more components should be in that presentation, I can set it to true no problem.

Reply:
I also think that the default should be `true`, but for this example, I think it's okay to set it to false.